hastexo/management/commands/undertaker.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings as django_settings
+from apscheduler.schedulers.blocking import BlockingScheduler
+
+from hastexo.utils import SETTINGS_KEY, DEFAULT_SETTINGS
+from hastexo.jobs import UndertakerJob
+
+
+class Command(BaseCommand):
+    help = 'Deletes stacks automatically'
```

Comment:
Perhaps change this to "Automates the deletion of stale stacks, i.e. those that have not been resumed in a configurable time period" to be a little more descriptive.